src/DGtal/images/ImageContainerByITKImage.ih

```diff
@@ -152,6 +152,20 @@ namespace DGtal
       return it.Get();
     }
 
+    template <typename TDomain, typename TValue>
+    inline
+    DGtal::ImageContainerByITKImage<TDomain,TValue>&
+    DGtal::ImageContainerByITKImage<TDomain,TValue>
+    ::operator=(const ImageContainerByITKImage& other)
+    {
+      if (this != &other)
+        {
+          myITKImagePointer = other.myITKImagePointer;
+          myDomain = other.myDomain;
```

Comment:
Should `updateDomain` be called here instead of copying domain, like in the copy constructor ?